tests/unit/lms/services/mailchimp_test.py

```diff
@@ -61,25 +61,6 @@ def test_it(self, mailchimp_transactional, caplog):
             }
         )
 
-    def test_if_theres_no_api_key_doesnt_call_mailchimp(self, mailchimp_client):
-        svc = MailchimpService(None)
-
-        svc.send_template(
-            sentinel.template_name,
-            EmailSender(
-                sentinel.subaccount_id,
-                sentinel.from_email,
-                sentinel.from_name,
-            ),
-            EmailRecipient(
-                sentinel.to_email,
-                sentinel.to_name,
-            ),
-            {},
-        )
-
-        mailchimp_client.messages.send_template.assert_not_called()
-
     @pytest.fixture
     def mailchimp_client(self, mailchimp_transactional):
         return mailchimp_transactional.Client.return_value
```

Comment:
I reckon one of this fixtures is no longer used and coverage fails.

Reply:
Fixed